SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialClassInfoManager.cpp

```diff
@@ -292,6 +292,51 @@ UClass* USpatialClassInfoManager::GetClassByComponentId(Worker_ComponentId Compo
 	return nullptr;
 }
 
+uint32 USpatialClassInfoManager::GetComponentIdForClass(const UClass& Class) const
+{
+	const FString ClassPath = Class.GetPathName();
+	if (const FSchemaData* SchemaData = SchemaDatabase->ClassPathToSchema.Find(ClassPath))
+	{
+		return SchemaData->SchemaComponents[SCHEMA_Data];
+	}
+	return SpatialConstants::INVALID_COMPONENT_ID;
+}
+
+TArray<Worker_ComponentId> USpatialClassInfoManager::GetComponentIdsForClass(const UClass& BaseClass, const bool bIncludeDerivedTypes /* = true */) const
```

Comment:
I think the name of this function is a bit misleading since it's giving component ids not just for the class in question but also all the subclasses.

Reply:
Agree. I went back and forth on the name. You convinced me. :)